lib/sycamore/sycamore/utils/pdf.py

```diff
@@ -125,12 +125,18 @@ def convert_from_path_streamed_batched(filename: str, batch_size: int) -> Genera
         yield batch
 
 
-def pdf_to_image_files(pdf_path: str, file_dir: Path) -> Generator[Path, None, None]:
+def pdf_to_image_files(pdf_path: str, file_dir: Path, fh = None, pages = None) -> Generator[Path, None, None]:
     """Writes the files (streamed) into file_dir.  Caller is responsible for calling
     path.unlink() to cleanup the files.
 
     Note: model service will call this to get images for processing"""
 
+    if pdf_path == '-' and fh is None:
```

Comment:
what's this for? we can just make the arg optional?

Reply:
https://linux.die.net/man/1/pdftoppm - pdftoppm accepts '-' as stdin which is a standard thing unix utilities accept.

Reply:
Interesting, til